Makefile

```diff
@@ -408,38 +408,30 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=0.11.0
+protoVer=0.11.2
 protoImageName=ghcr.io/cosmos/proto-builder:$(protoVer)
```

Comment:
Run `docker build --pull --rm -f "contrib/devtools/Dockerfile" -t cosmossdk-proto:latest "contrib/devtools"` and replace this line by `cosmossdk-proto:latest` for testing locally.

Reply:
I would love to have someone using macOS try this.

Reply:
testing on my Mac now

Reply:
locally all works for me. Ran all the commands, thanks for this

Reply:
Can you check the permission of the api folder and the proto/tendermint folder? It should be your username and not root.